layouts/partials/tw/nav.html

```diff
@@ -1,137 +1,269 @@
+  >
+    Journalism that gets results for Pennsylvania
+  </p>
+  <ul
+    class="-order-1 hidden text-xs font-extrabold leading-tight tracking-wide sm:block sm:w-32 lg:w-52 lg:text-sm xl:text-base"
+  >
+    <li>Independent</li>
+    <li>Nonpartisan</li>
+    <li>Nonprofit</li>
+  </ul>
+</div>
+<nav
+  x-data="{ toggle: false }"
+  class="
+    relative flex h-11 w-full items-stretch justify-between border-b border-t bg-white font-sans md:justify-center
+  "
+  :class="toggle ? 'px-5 sm:px-3' : 'px-3'"
+>
+  <a
+    href="/search/"
+    class="z-20 flex flex-[0_1_70px] flex-row-reverse items-center justify-between gap-1 text-sm font-bold leading-none tracking-wide"
+    data-ga-category="search"
+    :data-ga-action="'modal:search:open'"
+    x-data
+    @click.prevent="$dispatch('modal:search:open')"
+  >
+    <svg class="h-[14px] w-[14px]" aria-label="Search">
+      {{ partial "helper/svg.html" "search" }}
+    </svg>
+    Search
+  </a>
+
+  <button
+    class="
+      no-js:hidden group z-20 order-1 flex flex-[0_1_70px] flex-row items-center gap-1 self-stretch text-sm font-bold leading-none tracking-wide
+    "
+    :class="toggle ? 'sm:mr-3 sm:-ml-3' : 'mx-0'"
+    type="button"
+    x-ref="button"
+    aria-label="Site Menu"
+    :aria-expanded="toggle"
+    :aria-controls="$id('dropdown-button')"
+    @click="toggle = ! toggle"
+    data-ga-category="burger"
+    data-ga-action="menu:toggle"
+    x-report-click
+  >
+    <span class="sm:hidden">Menu</span>
+    <span class="max-sm:hidden">More</span>
+    <div class="flex h-5 w-5 cursor-pointer items-center justify-center">
+      <div class="space-y-1">
+        <span
+          class="
+            block h-[2px] w-5 origin-center rounded-full bg-black transition-transform ease-in-out
+          "
+          :class="toggle ? 'translate-y-1.5 rotate-45' : ''"
+        ></span>
+        <span
+          class="
+            block h-[2px] w-5 origin-center rounded-full bg-black transition-transform ease-in-out
+          "
+          :class="toggle ? 'w-0 opacity-0' : 'opacity-1'"
+        ></span>
+        <span
+          class="
+            block h-[2px] w-3 origin-center rounded-full bg-black transition-transform ease-in-out
+          "
+          :class="toggle ? 'w-5 -translate-y-1.5 -rotate-45' : ''"
+        ></span>
+      </div>
+    </div>
+  </button>
+  <ul
+    class="
+      -top-2 right-1 z-10 ml-1 box-content flex-wrap gap-2 bg-white shadow sm:relative sm:top-auto sm:flex sm:h-auto sm:w-auto sm:max-w-4xl sm:flex-1 sm:flex-nowrap sm:items-center sm:justify-evenly sm:rounded-none sm:border-none sm:bg-transparent sm:bg-none sm:px-16 sm:shadow-none sm:before:content-[] md:overflow-visible md:px-[6.75rem]
+    "
+    :class="toggle ? 'max-sm:absolute max-sm:border rounded border-s-5 flex w-auto h-auto px-4 pt-14 pb-10 sm:pt-0 sm:pb-0 sm:overflow-visible' : 'overflow-hidden w-0'"
+    x-on="toggle"
```

Comment:
How does this work? My understanding is that x-on adds an event listener, but toggle is boolean. Is it just a typo of x-show or am I missing some deeper feature of Alpine?

Reply:
> How does this work? My understanding is that x-on adds an event listener, but toggle is boolean. Is it just a typo of x-show or am I missing some deeper feature of Alpine?

This is how I understand it as well. I could've named `toggle` anything. x-on is just less opinionated than x-show

Reply:
I think you can just drop the line then, no?